src/main/features/claim/form/models/eligibility/eligibility.ts

```diff
@@ -0,0 +1,90 @@
+import { Serializable } from 'models/serializable'
+import { YesNoOption } from 'models/yesNoOption'
+import { IsIn } from 'class-validator'
+import { ValidationErrors } from 'features/validationErrors'
+import { ValidationGroups } from 'claim/helpers/eligibility/validationGroups'
+import { ClaimValue } from 'claim/form/models/eligibility/claimValue'
+
+export class Eligibility implements Serializable<Eligibility> {
+
+  @IsIn(YesNoOption.all(), { message: ValidationErrors.YES_NO_REQUIRED, groups: [ValidationGroups.CLAIMANT_ADDRESS] })
+  claimantAddress?: YesNoOption
```

Comment:
why this is optional?

Reply:
because there's ~6 fields on this class and they populated at different times